internal/build/version.go

```diff
@@ -21,3 +32,13 @@ var Sha string
 
 // Version allows for a queryable version set at build time.
 var Version string
+
+// PrintBuildInfo prints the build information.
+func PrintBuildInfo() string {
+	buildInfo := &BuildInfo{Branch, Sha, Version}
+	out, err := json.Marshal(buildInfo)
```

Comment:
Suggest that we do yaml (or manual marshaling) here?

```
$ contour version
branch: version-cmd
sha: 51348913a93d32b0adae970110a850c681e51d3d
version: v1.2.0-90-g51348913
```

There reason being that there are no other Contour commands that emit JSON, and the YAML version is easier to crack with grep, awk, etc. whereas the JSON basically needs jq.

Reply:
@youngnick LMK what you think?

Reply:
Have updated to YAML

Reply:
Yeah, YAML is good.